### PR TITLE
Add enrollment status to grade report

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3671](https://openedx.atlassian.net/browse/DOC-3671)

The grade report has a new column for enrollment status, added under EDUCATOR-435. I added the new column to the report description and updated the grade report screenshot with one that includes the new column. 

### Date Needed

This feature has been merged, so we should get the documentation updated soon.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @iloveagent57 
- [x] Subject matter expert: @sanfordstudent 
- [x] Doc team review (copy edit): @edx/doc

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/student_progress/course_grades.rst [hunk 1]
```diff
@@ -242,6 +242,9 @@ columns that provide the following information.
   eligible for a certificate, or if the certificates for a course have not yet
   been generated, the value in this column is "N/A".
 
+* The **Enrollment Status** column indicates whether the learner is enrolled
```
Comment (iloveagent57) [inline en_us/shared/student_progress/course_grades.rst]: I would maybe specify that it indicates whether the learner is *currently* enrolled or unenrolled in the course.  Any unenrolled learner included in the report for a course was at one point enrolled in the course.

**Review by iloveagent57 — COMMENTED** [review]
Looks good, one minor suggestion.

**Review by iloveagent57 — APPROVED** [review]
👍

**Comment by catong** [comment]
👍

**Review by sanfordstudent — APPROVED** [review]

**Pull request merged**